CenterForOpenScience/osf.io
# [ENG-2121] Add node storage limits to model

## Purpose

Creates an indicator property that shows when a node is over or at it's storage limit.

## Changes

- adds field for custom  storage limits
- add StorageUsage enum with methods for determining status
- adds tests

## QA Notes

  - Does this change require a data migration? If so, what data will we migrate?
small auto-migration
  - What is the level of risk?
Low low
    - Any permissions code touched?
No
    - Is this an additive or subtractive change, other?
addtive
  - How can QA verify? (Through UI, API, AdminApp or AdminAdminApp?)
Dev test
  - What features or workflows might this change impact?
Not
  - How will this impact performance?
Not
-->

## Documentation

No new docs

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2121
